crates/bevy_core_widgets/src/callback.rs

```diff
@@ -27,7 +28,7 @@ use bevy_ecs::world::{DeferredWorld, World};
 /// // Later, when we want to execute the callback:
 /// app.world_mut().commands().notify(&callback);
 /// ```
-#[derive(Default, Debug)]
+#[derive(Default, Debug, Reflect)]
```

Comment:
Nit: While we're here we could also register `ReflectDefault` (i.e. `#[reflect(Default)]`), right? Same goes for the other files in this PR